src/Jellyfin.Drawing.Skia/StripCollageBuilder.cs

```diff
@@ -22,6 +23,9 @@ public StripCollageBuilder(SkiaEncoder skiaEncoder)
         _skiaEncoder = skiaEncoder;
     }
 
+    [GeneratedRegex("(/s{IsArabic}||/s{IsArmenian}||/s{IsHebrew}||/s{IsSyriac}||/s{IsThaana})+", RegexOptions.IgnoreCase)]
```

Comment:
```suggestion
    [GeneratedRegex("\p{IsArabic}|\p{IsArmenian}|\p{IsHebrew}|\p{IsSyriac}|\p{IsThaana}", RegexOptions.IgnoreCase)]
```

I think the regex supplied is incorrect

Also, I don't think `IgnoreCase` is required for these character groups? Though to be fair I don't know anything about RTL languages

Reply:
> I think the regex supplied is incorrect
> 
> Also, I don't think `IgnoreCase` is required for these character groups? Though to be fair I don't know anything about RTL languages

Yes, You are right there is no case for rtl languages

Reply:
I made some additional tests and there is something strange going one with testing for multiple groups at once. Will need more time to test that.

Reply:
Also, make sure to remove the hard-coded width

Reply:
I think if you move to the regex I suggested you would have more success

Reply:
> I think if you move to the regex I suggested you would have more success

I agree

Reply:
```
Microsoft (R) Visual C# Interactive Compiler version 4.4.0-6.22559.4 ()
Loading context from 'CSharpInteractive.rsp'.
Type "#help" for more information.
> var regex = new System.Text.RegularExpressions.Regex(@"\p{IsArabic}||\p{IsArmenian}||\p{IsHebrew}||\p{IsSyriac}||\p{IsThaana}");
> regex.IsMatch("test")
true
> regex.IsMatch("بْجَدِيَّة الْعَرَ")
true
> 
```
Nah not really. As i said, weird stuff happening i have to test beforehand.

Reply:
Dont mind me, i cannot read.

Reply:
I geuss it is related to the way you provided that arabic I can give you arabic words to test

Reply:
wait wait or is written like this in regex | not double

Reply:
copy and paste the text crobibero wrote

Reply:
you can try this
bool hasRTL = Regex.IsMatch("نص باللغة العربيه", @"\p{IsArabic}|\p{IsHebrew}|\p{IsSyriac}|\p{IsThaana}");

Reply:
you xan try this
`bool hasRTL = Regex.IsMatch("نص باللغة العربيه", @"\p{IsArabic}|\p{IsHebrew}|\p{IsSyriac}|\p{IsThaana}");`

Reply:
It works perfectly fine if using the Regex I sent.

https://dotnetfiddle.net/YkYPxB

Reply:
> Dont mind me, i cannot read.